packages/material-ui-lab/src/TimelineItem/TimelineItem.js

```diff
@@ -80,19 +84,21 @@ const TimelineItem = React.forwardRef(function TimelineItem(inProps, ref) {
 
   const styleProps = {
     ...props,
-    align,
+    position: positionProp || positionContext || 'left',
```

Comment:
```suggestion
    position: positionProp || positionContext || 'right',
```

Same here, isn't `right` the correct default?

Reply:
It's true. I'm fixing it